cpu/cortex-m0_common/include/core_cmFunc.h

```diff
@@ -342,7 +342,7 @@ __attribute__( ( always_inline ) ) __STATIC_INLINE void __disable_irq(void)
  */
 __attribute__( ( always_inline ) ) __STATIC_INLINE uint32_t __get_CONTROL(void)
 {
-  uint32_t result;
+  uint32_t result = -1;
 
   __ASM volatile ("MRS %0, control" : "=r" (result) );
```

Comment:
False positive, if I am not mistaken: This ASM operation writes the `control` special purpose register value to the register represented by %0, i.e. the `result` variable, which means that this variable is always written before `return(result);`
Therefore, it is safe to suppress static code warnings about uninitialized variables in this (and the following) ASM wrapper inlines.